ui/src/pages/Alert/Event/index.tsx

```diff
@@ -22,13 +22,22 @@ import type { ColumnsType } from 'antd/es/table';
 import dayjs from 'dayjs'
 import AlarmFilter from '../AlarmFilter';
 import { sortEvents } from '../helper';
+import RuleDrawerForm from '../Rules/RuleDrawerForm';
+import { useState } from 'react';
 const { Text } = Typography;
 
 export default function Event() {
   const [form] = Form.useForm();
+  const [drawerOpen,setDrawerOpen] = useState<boolean>(false);
+  const [editRuleName, setEditRuleName] = useState<string>();
   const { data: listAlertsRes, run: getListAlerts } = useRequest(
     alert.listAlerts,
   );
+  const editRule = (rule:string)=>{
```

Comment:
This file seems not to be formatted.

Reply:
Maybe I need a tool that automatically formats before submitting.

Reply:
Yes, you'd better add a step in husky hook before real commit.